idris-erlang.cabal

```diff
@@ -20,13 +20,9 @@ executable idris-codegen-erlang
   other-modules:       IRTS.CodegenErlang
                      , IRTS.CodegenErlang.Foreign
 
-  extensions:          PatternSynonyms
```

Comment:
Oh, this should enable these 3 extensions for all the files in the project (thus precluding the `{-# LANGUAGE ... #-}` comments), is this not the case? Maybe it's bad style anyway and I shouldn't have done this.